test/integration/controllermanager/shoot/maintenance/maintenance_test.go

```diff
@@ -358,6 +364,12 @@ var _ = Describe("Shoot Maintenance controller tests", func() {
 				g.Expect(testClient.Get(ctx, client.ObjectKeyFromObject(shoot), shoot)).To(Succeed())
 				g.Expect(*shoot.Spec.Provider.Workers[0].Machine.Image).To(Equal(gardencorev1beta1.ShootMachineImage{Name: highestAMD64MachineImage.Name, Version: highestAMD64MachineImage.Version}))
 				g.Expect(*shoot.Spec.Provider.Workers[1].Machine.Image).To(Equal(gardencorev1beta1.ShootMachineImage{Name: highestARM64MachineImage.Name, Version: highestARM64MachineImage.Version}))
+				g.Expect(shoot.Status.LastMaintenances).ToNot(BeNil())
+				g.Expect(shoot.Status.LastMaintenances[0].Operations).To(ConsistOf(
+					"Reconcile following resources: Infrastructure, DNSRecordInternal, DNSRecordExternal, DNSRecordIngress",
+					"Machine image of worker-pool \"cpu-worker1\" from \"foo-image\" version \"0.0.1-beta\" to version \"1.1.1\". Reason: MachineImage expired - force update required",
```

Comment:
Nit:
```suggestion
					ContainSubstring("Machine image of worker-pool %q from %q version %q to version %q. Reason: MachineImage expired - force update required", "cpu-worker1", "foo-image", "0.0.1-beta", "1.1.1"),
```
same for others.

Reply:
Is this required, for the test I think a static string should be fine.

Reply:
I don't have a strong opinion.